pinot-common/src/main/java/org/apache/pinot/common/function/scalar/ArithmeticFunctions.java

```diff
@@ -104,4 +111,30 @@ public static double ln(double a) {
   public static double sqrt(double a) {
     return Math.sqrt(a);
   }
+
+  @ScalarFunction
+  public static double sign(double a) {
+    return Math.signum(a);
+  }
+
+  @ScalarFunction
+  public static double pi() { return Math.PI; }
+
+  @ScalarFunction
+  public static double e() { return Math.E; }
+
+  @ScalarFunction
+  public static double power(double a, double b) {
+    return Math.pow(a, b);
+  }
+
+  @ScalarFunction
+  public static double log(double a) {
```

Comment:
We can also add `log2`
```suggestion
  public static double log10(double a) {
```

Reply:
This function should actually take a base as an optional parameter to be sql compliant  (I understand that this mechanism doesn’t support optional parameters, but a default value could possibly be used.)

If the base parameter is default, a natural logarithm should be computed. If it is 10 or 2, the appropriate math functions should be called. If it is something else it should be calculated as follows: 

logb(x) = log10(x) / log10(b)

Reply:
Having said that, there are log10 and log2 in various databases, so we should add these. It seems not all databases allow arbitrary bases either.

Reply:
Postgres accepts a base https://database.guide/how-log-works-in-postgresql/ but sql server accepts the base in a different parameter position https://www.w3schools.com/sql/func_sqlserver_log.asp

Maybe we can just ignore bases and add log (natural), log2 and log10.

Reply:
I referenced the presto functions: https://prestodb.io/docs/current/functions/math.html

Reply:
Yeah, we don't need to strictly follow presto, and I just put it as a reference. Actually presto does not support `log`, postgres and sql server seems having opposite behavior, so let's pick one, or skip it for now

Reply:
I have gone with ln, log2 and log10 following presto. I can add a variant as well allowing base parameter.